api/tests/pokemon.test.js

```diff
@@ -47,5 +48,16 @@ describe('GET /pokemon', () => {
   it('should accept a limit as an integer', async () => {
     const pokemon = await Posts.getPokemans(path, '40')
     expect(pokemon.body.results.length).toBe(40)
+  }) 
+  // Tried to test the "Count" key's value ->
```

Comment:
as you see from the below feedback, we should strive to have our code be as naturally readable as possible, such that code comments are unnecessary :)